src/components/Footer/components/Copyright.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Typography, Link } from "@mui/material";
+
+export default function Copyright() {
+  
+
+  return (
+    <Typography variant="body2" color="text.secondary">
+     {`Copyright @ `}
```

Comment:
it is not interpolation

Reply:
i made interpolation in variable  fullDate
but i need this {` `} for whitespaces inside Typography